packages/dicomImageLoader/src/imageLoader/createImage.ts

```diff
@@ -283,7 +234,7 @@ function createImage(
           // If we don't need the RGBA but the decoding is done with RGBA (the case
           // for JPEG Baseline 8 bit color), AND the option specifies to use RGB (no RGBA)
           // we need to remove the A channel from pixel data
-          const colorBuffer = new Uint8ClampedArray(
+          const colorBuffer = new Uint8Array(
```

Comment:
Please add a comment about vtk needing the Uint8Array here - it wouldn't be good to change that back unknowing.